tests/test_objective_funs.py

```diff
@@ -99,14 +100,26 @@ def test(eq):
         test(Equilibrium(node_pattern="quad", iota=PowerSeriesProfile(0)))
         test(Equilibrium(node_pattern="quad", current=PowerSeriesProfile(0)))
 
+    @pytest.mark.unit
+    def test_target_iota(self):
+        """Test calculation of iota profile."""
+
+        def test(eq):
+            obj = TargetIota(target=1, weight=2, eq=eq)
+            iota = obj.compute(eq.R_lmn, eq.Z_lmn, eq.L_lmn, eq.i_l, eq.c_l, eq.Psi)
+            np.testing.assert_allclose(iota, -2 / 3)
```

Comment:
Actual iota is 0 but the target is 1, so the error is -1. Then with a weight of 2 I would expect the result to be -2. How do we get -2/3?

Reply:
This is the grid spacing thing I mentioned. The residual is `(f-target)*weight*spacing` and in this case the default grid has `L_grid` radial nodes (and `L_grid` is 3

Reply:
Ah OK. Sounds good.